crates/wash-cli/src/common/registry_cmd.rs

```diff
@@ -133,6 +133,7 @@ pub async fn registry_push(
                 || project_config.is_some_and(|c| c.common.registry.push_insecure),
             insecure_skip_tls_verify: cmd.opts.insecure_skip_tls_verify,
             annotations,
+            use_monolithic_push: cmd.use_monolithic_push,
```

Comment:
NIT: wonder if the `use_` is necessary here -- I'm always one for long meaningful/easy to understand names but... I think `monolithic_push` might be fine

Reply:
Fair, I've switched the internal flag to `monolithic_push`, should be fine either way I think, appreciate the feedback!